crates/bevy_render/src/view/window.rs

```diff
@@ -244,18 +270,27 @@ pub fn prepare_windows(
             let frame = surface
                 .get_current_texture()
                 .expect("Error configuring surface");
-            window.swap_chain_texture = Some(TextureView::from(frame));
+            window.swap_chain_texture_view = Some(TextureView::from(
+                frame.texture.create_view(&Default::default()),
+            ));
+            window.swap_chain_texture = Some(SurfaceTexture::from(frame));
```

Comment:
Not super important, but this should probably be a method on window since it's repeated 3 times.